src/configurations/destinations/accoil_analytics/ui-config.json

```diff
@@ -0,0 +1,196 @@
+{
+  "uiConfig": {
+    "baseTemplate": [
+      {
+        "title": "Initial setup",
+        "note": "Review how this destination is set up",
+        "sections": [
+          {
+            "groups": [
+              {
+                "title": "Connection settings",
+                "note": "Update your connection settings here",
+                "icon": "settings",
+                "fields": [
+                  {
+                    "type": "textInput",
+                    "label": "API Key",
+                    "configKey": "apiKey",
+                    "regex": "^[0-9a-zA-Z_]{1,80}$",
+                    "regexErrorMessage": "Invalid API Key",
+                    "placeholder": "Your Accoil.com API Key. You can find your API Key in your Accoil account settings.",
```

Comment:
placeholder should be different, It should be something like e.g. 5fXXXX0. 
and add note like this `"note": "Your Accoil.com API Key. You can find your API Key in your Accoil account settings."`

Reply:
I've switched the placeholder to note and put an eg